src/components/Nav.svelte

```diff
@@ -1,75 +1,136 @@
 <script>
-  export let segment;
+  import MobileMenu from "./MobileMenu.svelte";
 </script>
 
 <style>
-  nav {
-    border-bottom: 1px solid rgba(255, 62, 0, 0.1);
-    font-weight: 300;
-    padding: 0 1em;
+  header {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    padding: 12px 0;
+    background-color: #fff;
+    border-bottom: 1px solid hsl(0, 0%, 66.7%);
+    z-index: 1;
+  }
+
+  img {
```

Comment:
I suppose Svelte would automatically scope the styles, but it's better to use explicit class name/ID instead of tag type, so maintainer can see something like `.logo {` and know which element is the style intended for.